ebill_paynet_customer_free_ref/models/sale_order.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    @api.depends("customer_order_number")
+    def _compute_paynet_client_order_ref(self):
+        for order in self:
```

Comment:
No call to super in this? Maybe add a comment to state why.

Reply:
No call  to super because the value retrieved by the base implementation is made obsolete by this module.